buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -437,8 +437,10 @@ def set_sdk_variables() {
     DATESTAMP = get_date()
     SDK_FILENAME = "OpenJ9-JDK${SDK_VERSION}-${SPEC}-${DATESTAMP}.tar.gz"
     TEST_FILENAME = "test-images.tar.gz"
+    JAVADOC_FILENAME = "OpenJ9-JDK${SDK_VERSION}-Javadoc-${SPEC}-${DATESTAMP}.tar.gz"
     echo "Using SDK_FILENAME = ${SDK_FILENAME}"
     echo "Using TEST_FILENAME = ${TEST_FILENAME}"
+    echo "Using JAVADOC_FILENAME = ${JAVADOC_FILENAME}"
```

Comment:
Minor nit, since we're echoing the debug output for `JAVADOC_FILENAME` we should also echo the value of `ARCHIVE_JAVADOC`.